package.json

```diff
@@ -32,11 +32,12 @@
     "@types/express": "4.11.1",
     "@types/helmet": "0.0.37",
     "@types/jsonwebtoken": "7.2.6",
+    "@types/moment": "2.13.0",
```

Comment:
Can this commit be squashed with the commit that uses it so that it's easier to understand how/why it's used?